### DISCUSSION TITLE
Full Portable How to...

### DISCUSSION THREAD

**Discussion by majidi-mehdi** [Eugeny/tabby#9043] [Q&A]
Hello friends,
How can I use the program completely portable?
How can I change the data path?
thank you

**Comment by Ph0b054n0m4ly** [comment]
I'm wondering the same thing.  The latest on their github page is:

"Tabby will run as a portable app on Windows, if you create a data folder in the same location where Tabby.exe lives"

I've done those steps, but it still creates a config.yaml file in "C:\Users\USERNAME\AppData\Roaming\tabby\"

**Comment by Strike9i** [comment]
Is nobody using Tabby portable? Since it is jan 2025 now, and i have the same problem in the 1.0.220 version. The portable version in the .zip file creates a config.yaml in directory in AppData\Roaming\tabby and it ignores the "--config" flag. What is the alternative to use tabby accross pc's?
